crates/ruff_cli/Cargo.toml

```diff
@@ -4,9 +4,9 @@ version = "0.0.272"
 authors = ["Charlie Marsh <charlie.r.marsh@gmail.com>"]
 edition = { workspace = true }
 rust-version = { workspace = true }
-documentation = "https://github.com/charliermarsh/ruff"
-homepage = "https://github.com/charliermarsh/ruff"
-repository = "https://github.com/charliermarsh/ruff"
+documentation = "https://github.com/astral-sh/ruff"
```

Comment:
Should we use https://beta.ruff.rs/docs/configuration/#command-line-interface here?

Reply:
I'd link to the top level docs in all crates, mainly since e.g. the cli crate is the entrypoint for maturin and iirc uses those links for pypi

Reply:
PyPi uses the links from `pyproject.toml`. I think this should be used by https://docs.rs if the crate was published.